src/OrchardCore/OrchardCore.Navigation.Core/NamedNavigationProvider.cs

```diff
@@ -0,0 +1,30 @@
+namespace OrchardCore.Navigation;
+
+public abstract class NamedNavigationProvider : INavigationProvider
```

Comment:
IMHO no need to introduce a new type, `AdminNavigationProvider` is enough. We can have `Name` as an overridable property, so if anyone wants to use a other name than `Admin` he could

Reply:
`NamedNavigationProvider` is much cleaner where any one that want to create a base class can.

Reply:
You can check the Orchard 1, no need to add more base classes IMHO

Reply:
I don't see anything wrong with the `NamedNavigationProvider` in this case.

Reply:
> public abstract class AdminMenuNavigationProvider : NamedNavigationProvider

Reply:
I like to keep the below: Also Seb is also okay with it. 

```
AdminNavigationProvider : NamedNavigationProvider
```